stable/datacube-wps/values.yaml

```diff
@@ -30,13 +30,14 @@ wps:
   annotations: { }
   additionalEnvironmentVars: { }
   livenessProbe:
-    httpGet:
-      path: /?service=WPS&request=GetCapabilities&version=1.0.0
-      port: http
-  readinessProbe:
-    httpGet:
-      path: /?service=WPS&request=GetCapabilities&version=1.0.0
-      port: http
```

Comment:
Removing readiness probe check is fine in this context (since datacube-chart is just a template file). However, both liveness probe and readiness probe should be used within service configuration file.

Reply:
This is been modified to match ows current setting as wps is merging in to ows in the future, all settings are configured in ows.yaml and it has been updated to reflect this upcoming version.